project/Dependencies.scala

```diff
@@ -60,9 +61,9 @@ object Dependencies {
 
     val slf4jApi = "org.slf4j" % "slf4j-api" % slf4jVersion // MIT
 
-    // mirrored in OSGi sample https://github.com/akka/akka-samples/tree/master/akka-sample-osgi-dining-hakkers
-    val osgiCore = "org.osgi" % "org.osgi.core" % "4.3.1" // ApacheV2
-    val osgiCompendium = "org.osgi" % "org.osgi.compendium" % "4.3.1" // ApacheV2
+    // mirrored in OSGi sample https://github.com/akka/akka-samples/tree/2.5/akka-sample-osgi-dining-hakkers
+    val osgiCore = "org.osgi" % "org.osgi.core" % "6.0.0" // ApacheV2
```

Comment:
To doublecheck.

Perhaps we could even consider excluding the OSGi module from our binary compatibility guarantees?